src/cargo/ops/cargo_compile.rs

```diff
@@ -474,6 +474,8 @@ fn scrape_target_config(config: &Config, triple: &str)
                     let list = value.list(&k)?;
                     output.cfgs.extend(list.iter().map(|v| v.0.clone()));
                 }
+                // FIXME: what happens with "warning" and "rerun-if-changed",
+                // should they really go into metadata?
```

Comment:
So `BuildOutput::metadata` from `./cargo/config` isn't used anywhere later on? Why is it gathered then?